index.html

```diff
@@ -269,5 +254,68 @@ <h3 class="about-title">Skills</h3>
           </div>
         </div>
       </section>
+
+      <section id="contact-form">
+        <div class="contact-form">
+
+          <div class="title">
+           <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+          </div>
+
+          <form action="https://formspree.io/f/xwkjdndo" class="contact-container" method="post">
+
+              <ul>
+                <li class="full-name">
+                  <input type="text" id="full-name" name="full-name" placeholder="Full name" maxlength="30" aria-label="Full name" required>
+                </li>
+                
+                <li class="first-name">
+                  <input type="text" id="first-name" name="first-name" placeholder="First name" maxlength="30" pattern="[a-zA-Z]" aria-label="First name" required>
+                </li>
+
+                <li class="last-name">
+                  <input type="text" id="last-name" name="last-name" placeholder="Last name" maxlength="30" pattern="[a-zA-Z]" aria-label="Last name" required>
+                </li>
+
```

Comment:
- [ ] nice job so far. Kindly fix the contact section such that it matches the Figma design. You can omit the last name and use just the first name
- [ ] please, kindly align the get in touch button as it appears on the Figma version.
- [ ] you can see your version is missing the background for the contact section. Kindly add that so the two designs can match. Remember, this helps you learn how to reproduce designs. 😄 
- [ ] please kindly add the height of the contact section
your design
![Screenshot (1123)_LI](https://github.com/Jtaborda21/My-Portfolio/assets/65298048/42cc7a36-f8fa-4deb-a0bd-8593587decc8)
original
![Screenshot (1122)](https://github.com/Jtaborda21/My-Portfolio/assets/65298048/27d41f19-bfe1-448f-982b-4328da3943d6)